src/vm/compilestubs.cpp

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
I have submitted PR https://github.com/dotnet/coreclr/pull/2024 that should make these stubs unnecessary.